VisualPinball.Unity/VisualPinball.Unity/VPT/Table/TableBehavior.cs

```diff
@@ -68,6 +68,8 @@ public class TableBehavior : ItemBehavior<Engine.VPT.Table.Table, TableData>
 		// note: this cache needs to be keyed on the engine material itself so that when its recreated due to property changes the unity material
 		// will cache miss and get recreated as well
 		private readonly Dictionary<PbrMaterial, UnityEngine.Material> _unityMaterials = new Dictionary<PbrMaterial, UnityEngine.Material>();
+		// keep a list of texture names that need recreation, serialized and lazy so when undo happens they'll be considered dirty again

```

Comment:
Not extremely relevant, but when you document a field, I would prefer the triple slashes. They better separate the member and also show up in the editor hover documentation:

```cs
/// <summary>
/// Keeps a list of texture names that need recreation, serialized and
/// lazy so when undo happens they'll be considered dirty again.
/// </summary>
[SerializeField] 
private List<string> _dirtyTextures = new List<string>();
```